tests/testthat/test-tmb-methods.R

```diff
@@ -378,11 +386,6 @@ test_that("h_construct_model_frame_inputs works with all columns", {
     FEV1 ~ RACE + USUBJID + AVISIT,
     ignore_attr = TRUE
   )
-  expect_equal(
```

Comment:
why is this deleted? does it not work anymore?

Reply:
`h_construct_model_frame_inputs` does not need that many attributes anymore thus removed